clang/test/Sema/pragma-clang-section-coff.c

```diff
@@ -0,0 +1,7 @@
+// RUN: %clang_cc1 -fsyntax-only -verify %s -triple arm64-windows-msvc
+// expected-no-diagnostics
+#pragma clang section bss = "mybss.1" data = "mydata.1" rodata = "myrodata.1" text = "mytext.1"
+#pragma clang section bss="" data="" rodata="" text=""
+#pragma clang section
```

Comment:
I just compared with `clang/test/Sema/pragma-clang-section-macho.c`, that has some invalid examples in it. Do you have any invalid cases for coff?

Reply:
Solved with an aditional RUN command in the existing test